src/lib/Behat/BusinessContext/RolesContext.php

```diff
@@ -99,19 +99,46 @@ public function thereIsAPolicy(string $moduleAndFunction, string $limitation, st
             if (
                 $policy['Module'] === $expectedModule &&
                 $policy['Function'] === $expectedFunction &&
-                strpos($policy['Limitations'], $limitation) !== false
+                false !== strpos($policy['Limitations'], $limitation)
             ) {
                 $policyExists = true;
             }
         }
 
         if (!$policyExists) {
-            throw new Exception(sprintf('Policy "%s" with limitation "%s" not found.', $moduleAndFunction, $limitation));
+            throw new Exception(sprintf('Policy "%s" with limitation "%s" not found on the "%s" policies list.', $moduleAndFunction, $limitation, $roleName));
         }
     }
 
     /**
-     * @Then There's an assignment :limitation for :userOrGroup on the :roleName assignments list
+     * @Then there is no policy :moduleAndFunction with :limitation limitation on the :roleName policies list
+     */
+    public function thereIsNoPolicy(string $moduleAndFunction, string $limitation, string $roleName): void
+    {
+        $rolePage = PageObjectFactory::createPage($this->utilityContext, RolePage::PAGE_NAME, $roleName);
+        $rolePage->navLinkTabs->goToTab($this->tabMapping['policy']);
+        $adminList = $rolePage->adminLists[$this->tabMapping['policy']];
```

Comment:
Can we extract it to a separate method? Something like `isRoleWithLimitationPresent` in RolePage? Same thing is used in Step above.